plugins/cloud-azure/src/test/java/org/elasticsearch/cloud/azure/AbstractAzureRepositoryServiceTestCase.java

```diff
@@ -38,22 +38,7 @@
 
 public abstract class AbstractAzureRepositoryServiceTestCase extends AbstractAzureTestCase {
```

Comment:
So I checked in master branch and we are in a pretty good shape. We have 2 choices here:

* Remove the tests in 2.4 branch
* Keep them (but as they are not ran by default it's harmless IMO)

I'll go for the later, which means status quo on 2.4 branch if no one objects.